src/scripts/main.js

```diff
@@ -1,3 +1,354 @@
 'use strict';
 
-// write code here
+const mainPage = document.querySelector('body');
+
+let table = document.querySelector('tbody');
+
+const header = document.querySelector('thead').lastElementChild;
+
+let rows = [];
+
+const successTitle = 'Success';
+const successMessage = 'New employee record has been added successfully';
+const errorTitle = 'Error';
+let errorMessage = '';
+
+function sortAsc(i) {
+  rows = [...document.querySelector('tbody').children].sort((a, b) =>
```

Comment:
you can create variable  for `...document.querySelector('tbody')` and use it in your function, also you can use `[...table.tBodies][0]` and take `rows` from it